src/Aspire.Dashboard/Configuration/ValidateDashboardOptions.cs

```diff
@@ -90,12 +89,12 @@ public ValidateOptionsResult Validate(string? name, DashboardOptions options)
                             }
                             break;
                         default:
```

Comment:
There should be a case for null. It should say the value is required when a resource service is configured. See the switch statement for options.Otlp.AuthMode as an example.